README.md

```diff
@@ -55,21 +41,10 @@ Support can be found in one of the following places:
 - [Official support forum](https://namelessmc.com/forum)
 - [SpigotMC](https://www.spigotmc.org/threads/nameless-minecraft-website-software.34810/)
 
-Feature requests can be posted on the [forum](https://namelessmc.com/forum/view/7-web-feature-requests/), and bugs can be reported in the [GitHub Issues](https://github.com/NamelessMC/Nameless/issues) tab.
```

Comment:
Before we totally remove this link, we should get the ok from Sam, as I know he would like to keep the website somewhat involved.